packages/asset-swapper/src/utils/market_operation_utils/bridge_source_utils.ts

```diff
@@ -146,6 +148,15 @@ export function getCurveInfosForPair(chainId: ChainId, takerToken: string, maker
                             [makerToken, takerToken].filter(v => c.metaTokens?.includes(v)).length > 0),
                 ),
             );
+        case ChainId.Arbitrum:
+            return Object.values(CURVE_ARBITRUM_INFOS).filter(c =>
+                [makerToken, takerToken].every(
+                    t =>
+                        (c.tokens.includes(t) && c.metaTokens === undefined) ||
+                        (c.tokens.includes(t) &&
+                            [makerToken, takerToken].filter(v => c.metaTokens?.includes(v)).length > 0),
+                ),
```

Comment:
Not entirely relevant to this PR but this is logic is getting pretty duplicated, maybe we could DRY it up a bit 🤔 


```typescript
const tokenFilterFn = c => [makerToken, takerToken] t =>
  (c.tokens.includes(t) && c.metaTokens === undefined) ||
  (c.tokens.includes(t) && [makerToken, takerToken].filter(v => c.metaTokens?.includes(v)).length > 0);

...

case ChainId.Fantom
  return Object.values(CURVE_FANTOM_INFOS).filter(tokenFilterFn);
case ChainId.Arbitrum:
  return Object.values(CURVE_ARBITRUM_INFOS).filter(tokenFilterFn);

```